base_station/gui/src/components/LCMEcho.vue

```diff
@@ -82,6 +82,7 @@
           '/ra_pidconfig_cmd': false,
           '/rr_drop_complete': false,
           '/rr_drop_init': false,
+          '/rtcm': false,
```

Comment:
What is rtcm? Can we make this something more readable? I know other channels probably don't have the most readable names either but no need to continue bad habits.

Same goes for the LCM struct and adding at least a comment when rtcm gets introduced in the python program.

Reply:
[RTCM](https://www.rtcm.org/) stands for the Radio Technical Commission for Maritime Services of which have standardized the messages that a GPS use to coordinate with other devices. We use the RTCM messages that our base station GPS provides to acquire our RTK fix.

Similar to how we don't spell out the acronyms for "GPS", "IMU", or "LCM" I don't see why it would be valuable to name the channel for RTCM messages something other than RTCM. 

If you feel that it is necessary I could append "_msg" to the path, would that be sufficient?

Reply:
Ah okay. I still think a comment indicating this is for the base station GPS would be nice. Especially in the base station code where it is less obvious.